docs/aspnet4x-mvc.md

```diff
@@ -118,7 +118,7 @@ The `OktaMvcOptions` class configures the Okta middleware. You can see all the a
 | PostLogoutRedirectUri     | No           | The location Okta should redirect to after logout. If blank, Okta will redirect to the Okta login page. |
 | Scope                     | No           | The OAuth 2.0/OpenID Connect scopes to request when logging in. The default value is `openid profile`. |
 | LoginMode                     | No           | LoginMode controls the login redirect behavior of the middleware. The default value is `OktaHosted`. |
-| GetClaimsFromUserInfoEndpoint | No       | This property has been deprecated and will be no longer supported. |
+| GetClaimsFromUserInfoEndpoint | No       | Whether to retrieve additional claims from the UserInfo endpoint after login (not usually necessary). The default value is `true`. |
```

Comment:
nit: 🤔 If this is not usually necessary then why is the default `true`?

If that is intentional then it might be worth explaining why, like "The default value is true for compatibility with applications that require these claims" or something.

Reply:
This was not necessary in the past because we realized that since we were using the hybrid flow, all the claims were brought regardless of this `GetClaimsFromUserInfoEndpoint` value. That's why when we realized about this we decided to deprecate it. Now, this flag makes sense since we are using the code flow. This is how Okta.Asp.NetCore works too, so we are now consistent in both libraries.